SimpleSDMPolygons/src/operations/clip.jl

```diff
@@ -0,0 +1,42 @@
+function _get_polygon_from_bbox(bbox)
```

Comment:
i think a more general solution is to also have a method for a polygon from a list of coordinates

Reply:
added this as a constructor in SSP/src/types/geometry.jl